cmd/rook/rgw.go

```diff
@@ -54,7 +53,7 @@ func init() {
 }
 
 func startRGW(cmd *cobra.Command, args []string) error {
-	required := []string{"mon-endpoints", "cluster-name", "rgw-name", "rgw-host", "rgw-keyring", "public-ipv4", "private-ipv4"}
```

Comment:
I think it would be good to remove the rgw-name flag from the operator function that creates the container spec too, see https://github.com/rook/rook/blob/master/pkg/operator/object/ceph/rgw.go#L351.